README.md

```diff
@@ -1,3 +1,132 @@
 # ZMK Studio
 
-Initial work on the ZMK Studio UI.
+ZMK Studio is a powerful and user-friendly graphical interface for configuring and managing ZMK firmware for custom keyboards. It provides a seamless experience for users to customize their keyboard layouts, manage key bindings, and interact with their devices.
+
+## Overview
+
+ZMK Studio offers a range of capabilities to enhance the user experience:
+
+- **Graphical Keymap Editor**: Easily create and modify keymaps with a visual editor.
+- **Device Management**: Connect to your keyboard via USB or Bluetooth and manage its settings.
+- **Firmware Updates**: Flash new firmware to your device directly from the application.
+- **Real-time Feedback**: Receive real-time notifications and feedback from your device.
+
+For more detailed information on the features and capabilities of ZMK Studio, please visit the [ZMK Studio Documentation](https://zmk.dev/docs/features/studio#capabilities).
+
+## Getting Started
+
+To contribute to ZMK Studio, follow these steps:
+
+### Prerequisites
+
+Ensure you have the following installed on your system:
+
+- [Node.js](https://nodejs.org/) (version 22 or later)
+- [npm](https://www.npmjs.com/) (version 10 or later)
+- [Rust](https://www.rust-lang.org/tools/install) (for building Tauri applications)
+- [Tauri CLI](https://tauri.app/v1/guides/getting-started/prerequisites/)
```

Comment:
Let's link to v2 since that's what we use
```suggestion
- [Tauri CLI](https://v2.tauri.app/start/prerequisites/)
```
Also debatably don't need to list Rust then since Tauri's install pre-reqs have it listed... And then also maybe don't need to mention npm since Node.js installs it by default.

Reply:
done in https://github.com/zmkfirmware/zmk-studio/pull/114/commits/32c8fa465212453b1d1cef7991ed4f10d12e20a1